tests/README.md

```diff
@@ -0,0 +1,124 @@
+# tests
```

Comment:
Maybe a nicer, more descriptive heading like "MicroPython test suite"?

Reply:
Can do! Part of the reason I was vague here is that I wasn't actually sure this was the whole test suite... :)